src/main/java/step2/domain/LottoNumber.java

```diff
@@ -40,6 +40,11 @@ private boolean isValidRange(int number) {
 		return number >= LOTTO_MIN_NUM && number <= LOTTO_MAX_NUM;
 	}
 
+	@Override
+	public int compareTo(LottoNumber o) {
+		return this.number - o.number;
+	}
```

Comment:
만약 비교하고자 하는 값들이 음수라면 integer.Max_Value 값을 넘어 32비트가 초과될수 있기
때문에 오버플로우가 발생할수도 있기에 정수 값을 비교한다면 다음과 같이 변경하면 좋을 것 같습니다 :)
```suggestion
	@Override
	public int compareTo(LottoNumber o) {
		return Integer.compare(this.number, o.number);
	}
```